api-tests/server/settings_test.go

```diff
@@ -39,7 +39,7 @@ func TestSettings(t *testing.T) {
 		res, err := serverClient.Default.ServerService.GetSettings(nil)
 		require.NoError(t, err)
 		assert.True(t, res.Payload.Settings.TelemetryEnabled)
-		assert.True(t, res.Payload.Settings.SttEnabled)
```

Comment:
we need sanity check in Github CI to check that API tests can be built.

Reply:
This should be done within #2805 and it's out of scope of this ticket anyway. I had to make change because the code couldn't compile, but otherwise I didn't have to.

Reply:
Agree, just wrote it here to not forget